### DISCUSSION TITLE
[Feature Request] cheack box to search for all installed Items

### DISCUSSION THREAD

**Discussion by Gothdir** [BlafKing/sd-civitai-browser-plus#130] [Ideas]
Hey! Loving the browser! 
Sometimes I want to read the description of an Item and eventhrough you can save the description, they save as a giant wall of text because the formating doesnt get saved (some restriction of A1111, I guess). So it would be nive to show all installed Items with the click of a button with working filter. So I can see my installed checkpoints for ex.

Hope it can be done, thank for all the hard work!

**Comment by BlafKing** [comment]
Thanks for the kind words! glad to hear you like the extension :)

A feature already exists to load all the installed models.
In the update models tab, there is a "Load all installed models" button which you can use to load all the installed models from the selected content type(s)! :)

However, currently the filters don't have any effect on this, so I'll have a look if I can find a good way to implement this.

**Reply by hobolyra** [reply]
I would love for filtering installed models. That's the main reason I wanted to use this, but then find out I can't. D=
